test/k8s/simple_test.go

```diff
@@ -147,33 +149,42 @@ spec:
 	Context("scaling", func() {
 		var admin envoy_admin.Tunnel
 
+		var observer string
+		var observable string
+
 		BeforeAll(func() {
+			// finding a service for scaling (observable) and a service to observe the scale (observer)
+			for _, svc := range svcGraph {
```

Comment:
is it guaranteed that this exists? Wouldn't it be easier to deploy a fake-client that doesn't send requests?

Reply:
Yes, we pass 50 to the generator:
```go
svcGraph = graph.GenerateRandomServiceMesh(872835240, numServices, 50, instancesPerService, instancesPerService)
```
which means about half of all possible edges exist.

Reply:
I didn't find how to make `slow-cooker` to not send queries and just sleep. I guess it requires creating some other `fake-client` that does nothing. But technically each of the services is already such a type of "client".